README.md

```diff
@@ -40,7 +40,7 @@ Kinvey CLI is distributed as an NPM package. After you install NPM, run the foll
 
 * `flex init`
 
-   Configures Kinvey CLI to work with a specific Flex Service through prompts. This command is designed to be executed in a Node.js project directory where it creates a `.kinvey` configuration file. Each successive execution in the same directory overwrites the configuration file. Command-line options that specify the same data as the prompted-for data are ignored.
+   Configures Kinvey CLI to work with a specific Flex Service through prompts. This command is designed to be executed in a Node.js project directory where it creates a `.kinvey` configuration file. Information inside the file is saved per profile. Each successive execution in the same directory overwrites the respective profile section in the configuration file. Command-line options that specify the same data as the prompted-for data are ignored.
 
```

Comment:
Add that --email, --password, --instanceId options will be ignored or do you find it redundant?

Reply:
We've got this sentence at the end to cover just these: "Command-line options that specify the same data as the prompted-for data are ignored." Isn't it good enough?

Reply:
Not in this case because `flex init` doesn't prompt for credentials, anyways. It asks you to choose an app or an org and then, a service from the chosen app or org. But it expects that you are already authenticated. It used to accept email and password as option/env variables but since it starts keeping flex-related info per profile, we need a profile to be used with the command. Either active profile must be set or the command must be invoked with `--profile [profile-name-here]`.

Reply:
I see now. An update will follow.